apps/dashboard/web/polaris_web/web/src/apps/dashboard/components/shared/customDiffEditor.js

```diff
@@ -235,7 +226,34 @@ const transform = {
             headersMap: {...jsonObj.headersMap, ...payloadObj.headersMap},
             updatedData: jsonObj.updatedData,
         }
-    },  
+    },
+    formatData(data,style){
+        let localFirstLine = data?.firstLine
+        let finalData = ""
+        let payLoad = null
+        if(style === "http" && data && Object.keys(data).length > 0){
+            if(data.json){
+                Object.keys(data?.json).forEach((element)=> {
+                    if(element.includes("query")){
+                        if(data.json[element]){
+                            localFirstLine = localFirstLine + func.convertQueryParamsToUrl(data?.json[element])
+                        }
```

Comment:
This should result in the query params being added twice, since we are already adding them once before in the firstLine. Check again.